kafka-ui-react-app/src/components/KsqlDb/Query/ResultRenderer.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import { KsqlCommandResponse, Table } from 'generated-sources';
 
+import { ResultRendererWrapper } from './ResultRenderer.styled';
```

Comment:
In this project we use styleds like this:

Change `import { ResultRendererWrapper } from './ResultRenderer.styled';` to `import { * as S } from './ResultRenderer.styled';`

And rename `ResultRendererWrapper` to just `Wrapper`

Reply:
A do the same in other places that you touched

Reply:
Fixed this)